batchflow/models/torch/blocks/named_blocks.py

```diff
@@ -120,23 +112,77 @@ def __init__(self, inputs=None, layout='cnacn', channels='same', kernel_size=3,
             # If main flow changes the number of channels, so must do the side branch.
             # No activation, because it will be applied after summation with the main flow
             branch_params = {'layout': branch_layout, 'channels': channels[-1],
-                             'kernel_size': 1, 'stride': branch_stride_downsample}
+                             'kernel_size': 1, 'stride': branch_stride_downsample, 'groups': branch_groups}
         else:
-            branch_params = {}
+            branch_params = {'stride': branch_stride_downsample, 'groups': branch_groups}
         layout = 'R' + layout + op
 
         # Pass optional downsample parameters both to the main flow and to the side branch:
         # Only the first repetition is to be changed
         layer_params = [{'stride': stride_downsample,
-                         'branch': branch_params,
-                         'branch/stride': branch_stride_downsample}]
+                         'branch': branch_params}]
         layer_params += [{}]*(n_reps-1)
 
         super().__init__(*layer_params, inputs=inputs, layout=layout, channels=channels,
                          kernel_size=kernel_size, stride=stride, groups=groups, attention=attention,
                          **kwargs)
 
 
+class BottleneckBlock(ResBlock):
```

Comment:
Why is it a separate class?

Reply:
Also, we have much more files to replace `ResNet` with `bottleneck=True` option (e.g.: `densenet.py`, `resenet_with_stochastic_depth.py` , tutorial `00_project_from_scratch.ipynb`), please, don't forget about them.

Reply:
I can fix the tutorial, but not sure about everithing else:
`densenet` uses `DenseBlock`, not the `ResBlock`, `resnet_with_stochastic_depth` is written with tensorflow and better needs his own PR, all the other `bottleneck`s are about attention

Reply:
You are right, only tutorial needs changes.